src/app/View.ts

```diff
@@ -118,7 +112,30 @@ export class View {
     const containerGame = Render.elementFactory('div', {
       className: 'container-game',
     });
-    //render elements
+    const timerUpdate = setInterval(() => {
+      remainingTime.innerText = `Time Left : ${gameController.theTimer.theTurnTimeLeft} s`;
+      if (gameController.theTimer.theTurnTimeLeft === 0) {
+        alert('Your time is up!');
+        gameController.nextPlayer();
+        gameController.startTurn();
+      }
+    }, 1000);
```

Comment:
Here is something wrong, because we use interval every second, which contains another interval in countdownInterval (Timer.ts). I tried to solve this problem but I make some mistakes.